jbpm-human-task/jbpm-human-task-workitems/src/test/java/org/jbpm/services/task/wih/util/PeopleAssignmentHelperTest.java

```diff
@@ -55,39 +55,37 @@ public void testProcessPeopleAssignments() {
 		List<OrganizationalEntity> organizationalEntities = new ArrayList<OrganizationalEntity>();
 		
 		String ids = "espiegelberg,   drbug   ";
-		assertTrue(organizationalEntities.size() == 0);		
+		assertTrue(organizationalEntities.isEmpty());
 		peopleAssignmentHelper.processPeopleAssignments(ids, organizationalEntities, true);
 		assertTrue(organizationalEntities.size() == 2);
-		organizationalEntities.contains("drbug");
```

Comment:
organizationalEntities is List<OrganizationalEntities> so cannot contain String.